_posts/en/newsletters/2023-08-09-newsletter.md

```diff
@@ -0,0 +1,245 @@
+---
+title: 'Bitcoin Optech Newsletter #263'
+permalink: /en/newsletters/2023/08/09/
+name: 2023-08-09-newsletter
+slug: 2023-08-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Denial-of-Service (DoS) protection design philosophy:** Anthony
+  Towns [posted][towns dos] to the Lightning-Dev mailing list in
+  response to the [channel jamming][topic channel jamming attacks] part
+  of the notes for the recent LN developers meeting (see [Newsletter
+  #261][news261 jamming]).  The notes said "the cost that will deter an
+  attacker is unreasonable for an honest user, and the cost that is
+  reasonable for an honest user is too low [to deter] an attacker."
+
+  Towns suggested an alternative to trying to price out attackers: make
+  the costs paid by both attackers and honest users reflect the
+  underlying costs paid by the node operators providing the service.
+  That way, a node operator who is earning a reasonable return on
+  providing services to honest users will continue earning reasonable
+  returns if attackers begin using those services.  If the attackers try
+  to deny services to honest users by consuming an excess amount of node
+  resources, the nodes providing those resources have an
+  incentive---greater income---to scale up the resources they provide.
+
+  As a suggestion for how that scheme could work, Towns resurrected an
+  idea from several years ago (see [Newsletter #86][news86 hold fees])
+  about a combination of forward commitment fees and reverse hold fees
+  that would be charged in addition to the usual fee for a successful
+  payment.  As an HTLC propagated from spender Alice to forwarding node
+  Bob, Alice would pay Bob a small forward commitment fee; this would
+  correspond to Bob's cost for processing the HTLC (such as bandwidth).
```

Comment:
"this would correspond to Bob's cost for processing the HTLC"

But later we outline that the fee could also be used to compensate future participants along the route which, to me, makes this sentence confusing or incorrect (unless we include his forwarding fees as part of the processing of the HTLC, in addition to the bandwidth example)